tests_integration/__tests__/__snapshots__/early-exit.js.snap

```diff
@@ -1,5 +1,77 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`show detailed usage with --help config-precedence 1`] = `
+"Config Precedence (--config-precedence <cli-override|file-override|prefer-file>)
+
+  Define in which order config files and CLI options should be evaluated
+  cli-override  | default config => config file => CLI options
+  file-override | default config => CLI options => config file
+  prefer-file   | default config => config file (if config file is found) or
+                  default config => CLI options (if no config file is found)
```

Comment:
Should we remove these descriptions from `option.description`? I think they should belong to `choice.description`.